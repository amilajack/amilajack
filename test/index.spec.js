const path = require('path');
const childProcess = require('child_process');

describe('basic', () => {
  it('should render correctly', () => {
    const renderedAscii = childProcess.execSync('node amilajack.js', {
      cwd: path.join(__dirname, '..')
    });
    // Remove dynamically rendered text
    const asciiTextArray = renderedAscii.toString().split('\n');
    const filteredDynamicText = asciiTextArray.splice(asciiTextArray.length - 19);
    expect(filteredDynamicText).toMatchSnapshot();
  });
});
