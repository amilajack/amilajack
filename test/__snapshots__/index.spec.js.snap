// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic should render correctly 1`] = `
Array [
  "my personal w̶e̶b̶s̶i̶t̶e̶ npm module",
  "[90m┌──────────────────[39m[90m┬─────────────────────────────────────┐[39m",
  "[90m│[39m[31m Website          [39m[90m│[39m[31m Account                             [39m[90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m github           [90m│[39m https://github.com/amilajack        [90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m website/blog     [90m│[39m https://amilajack.com               [90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m email            [90m│[39m amilajack@gmail.com                 [90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m twitter          [90m│[39m https://twitter.com/amilajack       [90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m resume           [90m│[39m https://github.com/amilajack/resume [90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m paypal donations [90m│[39m https://paypal.me/amilajack         [90m│[39m",
  "[90m├──────────────────[39m[90m┼─────────────────────────────────────┤[39m",
  "[90m│[39m venmo donations  [90m│[39m https://venmo.com/amilajack         [90m│[39m",
  "[90m└──────────────────[39m[90m┴─────────────────────────────────────┘[39m",
  "",
]
`;
